src/odemis/acq/stream/_sync.py

```diff
@@ -812,25 +823,45 @@ def _estimateRawAcquisitionTime(self):
             res = self._sccd._getDetectorVA("resolution").value
             readout = numpy.prod(res) / ro_rate
 
-            exp = self._sccd._getDetectorVA("exposureTime").value
+            if hasattr(self._sccd, "integrationTime"):
+                exp = self._sccd.integrationTime.value
+                # calculate number of images to integrate
+                hwExpTimeMax = self._sccd.detector.exposureTime.range[1]
+                numImages2integrate = int(numpy.ceil(exp / hwExpTimeMax))
+                readout *= numImages2integrate
```

Comment:
Use the same code as in `_adjustHardwareSettings()`

Reply:
This should work? Would also simplify the line with the .exposureTime.

`            if hasattr(self._sccd, "integrationTime"):
                exp = self._sccd.integrationTime.value
                readout *= self._sccd.integrationCounts.value
            else:
                exp = self._sccd.exposureTime.value`

Reply:
Ah didn't process the comment on the code in `adjustHardwareSettings` yet.

Reply:
I think this code would work fine for the time estimation. (Moreover, you cannot really set the exposureTime on the hardware, so you may not be able to have exactly the same code as `adjustHardwareSettings()`.